spec/helpers/i18n_helper_spec.rb

```diff
@@ -26,57 +31,57 @@
     end
 
     it "sets the chosen locale" do
-      allow(helper).to receive(:params) { { locale: "es" } }
```

Comment:
Maybe we should create the method `provide_params` which takes a hash, puts it into the Parameters object and calls the `allow` part.

backportable too